state/zookeeper/zk.go

```diff
@@ -19,13 +19,17 @@ import (
 	"github.com/samuel/go-zookeeper/zk"
 )
 
-const defaultMaxBufferSize = 1024 * 1024
-const defaultMaxConnBufferSize = 1024 * 1024
+const (
+	defaultMaxBufferSize     = 1024 * 1024
+	defaultMaxConnBufferSize = 1024 * 1024
+)
 
 const anyVersion = -1
```

Comment:
add part of previous declarations ....

Reply:
This one was auto resolved, not sure why it didn't group them.